configuration/postgres/postgres_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package postgres
+
+import (
+	"context"
+	"regexp"
+	"testing"
+
+	"github.com/pashagolub/pgxmock"
+
+	"github.com/dapr/components-contrib/configuration"
+)
+
+func TestPostgresbuildQuery(t *testing.T) {
```

Comment:
Add test for select * without any conditions

Reply:
Will there be any negative cases?

Reply:
Added two scenarios for select * - where no keys are given and where no keys are provided by metadata is provided

Reply:
https://github.com/dapr/components-contrib/blob/90ba8a441d6ccc8987acf115ca4adf1f49fa9fba/configuration/postgres/postgres_test.go#L27-L47